doc/themes/the-things-stack/assets/js/theme.js

```diff
@@ -12,7 +12,33 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-document.addEventListener('DOMContentLoaded', function() {
+// When you change the domain, the easiest way to redirect people without downtime is to embed redirection script inside
+// the webpage, since it is delivered 'as a whole'
+
+(function(canonical, alternatives) {
+  if (!alternatives.some((alternative) => alternative.test(document.location.href))) {
+    let url = new URL(document.location.href)
+    window.location.replace(canonical + url.pathname + url.search + url.hash)
+  }
+})('https://www.thethingsindustries.com/docs', [
+  /^https:\/\/www.thethingsindustries.com\/docs/,
+  /^https:\/\/preview.thethingsindustries.com\/docs/,
+  /^http:\/\/localhost:1313/, // local development
+  /^https:\/\/[a-zA-Z0-9]+\.tk/, // .tk domains are useful for testing, no production service depends on this
+])
+
+document.addEventListener('DOMContentLoaded', function () {
+  migrationInfoAccepted = window.localStorage.getItem('migrationInfoAccepted')
+  if(migrationInfoAccepted != "true") {
+    let migrationBar = document.getElementById('migration-info')
+    migrationBar.style.display = 'flex';
+    let migrationButton = document.getElementById('migration-button')
+    migrationButton.addEventListener('click', function (){
+      window.localStorage.setItem("migrationInfoAccepted", true)
+      migrationBar.style.display = 'none';
+    })
+  }
```

Comment:
- Not declaring `var` on `migrationInfoAccepted`
- Mixing `'` and `"`
- Is `let` supported here or does it work in your modern browser?

Reply:
Yeah, I am not much of a web dev, so things like that happen to me. Fixed now.

To my defense, `let` is supported in all major browsers since September 2017 (Safari was latest to the party).